beta.py

```diff
@@ -62,27 +47,32 @@ def connect(url, hmac):
 				os.system('clear')
 			except:
 				os.system('cls')
-		r = requests.post(url, data=b64encode(cmd.encode("ascii")), headers={'Cookie':'session=' + hmac}, verify=False)
-		print(r.text)
+		#Error handling if issue posting
+		try:
+			r = requests.post(url, data=b64encode(cmd.encode("ascii")), headers={'Cookie':'session=' + hmac}, verify=False)
```

Comment:
As the headers don't appear to be changing too much (if at all), you could use a [requests.Session](https://requests.readthedocs.io/en/master/user/advanced/#session-objects) instead instead of spawning sessions on each call to requests.post